web/src/stories/button-group/base.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+
+import { Button } from '../../components/button'
+import { ButtonGroupProps, ButtonGroup } from '../../components/button-group'
+
+export default {
+  title: 'Components/Button Group',
+  component: ButtonGroup,
+}
+
+export const Base: React.FC<ButtonGroupProps> = props => (
+  <ButtonGroup {...props}>
+    <Button variant="primary">Button</Button>
+    <Button>Button</Button>
+    <Button appearance="minimal">Button</Button>
+  </ButtonGroup>
+)
+
+Base.parameters = {
+  design: {
+    type: 'figma',
+    url:
+      'https://www.figma.com/file/Mc03kFcaFlcMOA7afjaZmF/Grape-DS-Components-Web?node-id=385%3A6',
+  },
+}
```

Comment:
this can be moved into the default export (until we point to different figma files/nodes) seems to be the same for all!

Reply:
@tobiaslins The Figma url is different for every component.